src/gtkspellcheck/spellcheck.py

```diff
@@ -619,15 +661,32 @@ def _click_move_popup(self, *args):
 
     def _click_move_button(self, widget, event):
         if event.button == 3:
-            if self._deferred_check:
-                self._check_deferred_range(True)
-            x, y = self._view.window_to_buffer_coords(2, int(event.x), int(event.y))
-            iter = self._view.get_iter_at_location(x, y)
-            if isinstance(iter, tuple):
-                iter = iter[1]
-            self._marks["click"].move(iter)
+            self._move_mark_for_input(event.x, event.y)
         return False
 
+    def _move_mark_for_input(self, input_x, input_y):
+        if self._deferred_check:
+            self._check_deferred_range(True)
+        x, y = self._view.window_to_buffer_coords(2, int(input_x), int(input_y))
+        iter = self._view.get_iter_at_location(x, y)
+        if isinstance(iter, tuple):
+            iter = iter[1]
+        self._marks["click"].move(iter)
+
+    def __on_click_pressed(self, click, n_press, x, y) -> None:
```

Comment:
Is there a specific reason for using `__`?

The function may benefit from a more descriptive name (not that I did a particularly good job at naming the other functions). Maybe something like `_gtk4_on_textview_click`?

Reply:
Nope, that's an oops. In my own code I use privates everywhere and this one slipped through. Updated and renamed.